include_dir_impl/src/lib.rs

```diff
@@ -27,11 +29,14 @@ pub fn include_dir(input: TokenStream) -> TokenStream {
         panic!("\"{}\" doesn't exist", path.display());
     }
 
-    let path = path.canonicalize().expect("Can't normalize the path");
+    let path = path
+        .canonicalize()
+        .unwrap_or_else(|_| panic!("Can't normalize the path"));
 
-    let dir = Dir::from_disk(&path, &path).expect("Couldn't load the directory");
+    let entry = DirEntry::from_disk(&path, &path)
```

Comment:
Hang on... why are we changing the return type of `include_dir!()` from a `Dir` to a `DirEntry`? 

The main point of the `include_dir!()` macro is to embed a **directory** into your binary. I would also consider `DirEntry` as an implementation detail that gets exposed when traversing the directory tree. It's the `include_dir` analogue of [`std::fs::DirEntry`](https://doc.rust-lang.org/std/fs/struct.DirEntry.html).